gr-blocks/include/gnuradio/blocks/tagged_file_sink.h

```diff
@@ -57,8 +57,14 @@ class BLOCKS_API tagged_file_sink : virtual public sync_block
      * \param itemsize The item size of the input data stream.
      * \param samp_rate The sample rate used to determine the time
      *                  difference between bursts
+     * \param filename Ex: 'FileNameKeyword %%Y-%%m-%%d %%H:%%M:%%S.dat' 
+     *                 result as:
+     *                 'FileNameKeyword 2019-12-05 15:51:56.dat'.
```

Comment:
Can we make the filenames without spaces?

Reply:
That's just an example. The default value is an empty string, it isn't immediately clear what name that will result in. The default not having spaces would be a good thing.

Reply:
I'll fixup the example, I didn't like this space either.